packages/flutter_image/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.2
+
+- Migrates from `ui.hash*` to `Object.hash*`.
```

Comment:
Nit: Please use `*` as this file is already inconsistent, and `*` is our documented standard for files that aren't already consistently using `-`.

Reply:
fixed